src/app/docs/alert/alert-docs.component.html

```diff
@@ -4,10 +4,269 @@
+        </sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>
+          Text
+        </sky-docs-anatomy-item>
+        <sky-docs-anatomy-item
+          isOptional="true"
+        >
+          Close button
+        </sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Options"
+    >
+      
+        <sky-docs-design-guideline
+        heading="Alert type"
+      >
+        <table class="sky-docs-table">
+          <tr>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/alert/alert-type-1.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                Danger alert – for errors or to let the user know the system is in an unusable state.
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/alert/alert-type-2.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                Info alert – for messages that provide helpful information or context but do not require action.
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/alert/alert-type-3.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                Success alert – for messages conveying a positive or successful state.
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/alert/alert-type-4.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                Warning alert – for messages the user should know about and perhaps act on, but that do not constitute an error state or indicate that something is wrong.
+              </p>
+            </td>
+          </tr>
+        </table>
+      </sky-docs-design-guideline>
+        
+
+      <sky-docs-design-guideline
+        heading="Close button"
+      >
+        <p>
+          If the user should be able to dismiss the alert from the page after reading, you can include an optional close button.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/alert/alert-option-1.png"
```

Comment:
In this image, "pm" should be "p.m."

Also, this appears to be identical to `alert-usage-1.png` and `alert-type-2.png`. Should we just use one image for all instances?